hs_discover/src/components/Resources.vue

```diff
@@ -601,7 +627,7 @@ export default {
       margin: 0;
       transform: translateY(4px);
     }
-    .fa:hover {
+    .interactive:hover {
```

Comment:
This change to interactive... interactive doesn't seem to be toggled at all.  What is this for?

Reply:
That's how CSS works. This line controls when you hover over page left, for example, the icon changes color.

Reply:
but why the change from fa to interactive?  It looks like you just added interactive to everywhere that fa is used.

Reply:
There is one location fa is used and interactive is not: the hourglass in the left side of the search box.